### PR TITLE
Switch to jQuery AJAX request for group invite cancellation

### PR DESCRIPTION
**Pull request by WilsonChiang**
- This is part of a task to remove dependencies for prototype.js
- Also removed unused mapping.js file

### CODE REVIEW

File: app/views/assignments/_member.html.erb [hunk 1]
```diff
@@ -20,9 +20,8 @@ member.user.last_name %> <%= I18n.t("student.inviter") %></li>
    <% if grouping.is_inviter?(@current_user) %>
      <%= link_to I18n.t(:cancel), '#',
         onclick: "
-          new Ajax.Request('" +
-             disinvite_member_assignment_path(id: @assignment.id, membership: member.id) + "');",
-
+          jQuery.ajax({
+          url: '" + disinvite_member_assignment_path(id: @assignment.id, membership: member.id) +"'});",
```
Comment (david-yz-liu) [inline app/views/assignments/_member.html.erb]: This seems not to be indented properly. Also, use Ruby string interpolation.

**Comment by WilsonChiang** [comment]
@david-yz-liu is it ok to remove the prototype files now or are there other dependencies? Namely, `rails_pt_adapters.js` and `prototype-snippet.js`.

**Comment by david-yz-liu** [comment]
Probably still other dependencies.

**Comment by WilsonChiang** [comment]
Ok. That should've been the only AJAX request from prototype then.

**Pull request merged**